awx/ui_next/src/screens/Template/WorkflowJobTemplateAdd/WorkflowJobTemplateAdd.jsx

```diff
@@ -12,7 +12,19 @@ function WorkflowJobTemplateAdd() {
   const [formSubmitError, setFormSubmitError] = useState(null);
 
   const handleSubmit = async values => {
-    const { labels, organizationId, ...remainingValues } = values;
+    const {
+      labels,
+      inventory,
+      organization,
+      webhook_credential,
+      webhookKey,
+      ...remainingValues
```

Comment:
this would be better named postPayload or something...it was confusing to me that you were adding things to "remainingValues" below on first glance.